app/views/layouts/_top_menu.html.erb

```diff
@@ -15,8 +15,9 @@
           <li class="dropdown">
             <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button">People<span class="caret"></span></a>
             <ul class="dropdown-menu">
-              <%= nav_link "Police", police_people_path %>
-              <%= nav_link "CERT", cert_people_path %>
+              <% Department.managing_people.order("shortname ASC").each do |dept| %>
+                <%= nav_link dept.shortname, department_people_path(dept) %>
```

Comment:
unfortunately, this has the effect of omitting otherwise valid departments, if the department shortname is nil/empty.